pkg/probes/curl/curl_json_test.go

```diff
@@ -44,41 +44,45 @@ func TestCurlJSONProbe_GetExpandedUserData(t *testing.T) {
 		{
 			name: "happy path",
 			userDataVariables: map[string]string{
-				"TIMEOUT": "1",
-				"DELAY":   "2",
-				"URLS":    "http://example.com:80 https://example.org:443",
+				"TIMEOUT":      "1",
+				"DELAY":        "2",
+				"URLS":         "http://example.com:80 https://example.org:443",
+				"CURL_COMMAND": "curl",
```

Comment:
This doesn't actually fix the code, just the tests right? https://github.com/joshbranham/osd-network-verifier/blob/c1120adf883b0593eadc59413878e0ca61c04e21/pkg/probes/curl/curl_json.go#L105 runs and expects `presetUserVariables` to have `CURL_COMMAND` because `userdata-template.yaml` states `CURL_COMMAND` as required. You need to either modify the validation logic, or define the curl command earlier to satisfy the validation.